src/main/java/uk/gov/hmcts/darts/audio/helper/PostAdminMediasSearchHelper.java

```diff
@@ -128,8 +128,8 @@ private List<Predicate> addDateCriteria(CriteriaBuilder criteriaBuilder, Root<Me
         return predicateList;
     }
 
-    private String surroundWithPercentagesUpper(String value) {
-        return surroundValue(value.toUpperCase(Locale.ROOT), "%");
+    private String surroundWithPercentages(String value) {
+        return surroundValue(value, "%");
```

Comment:
Did you mean to remove the toUpperCase?

Reply:
yes, we are changing the queries to an `ilike`, which is case insensitive by default, so we don't need to uppercase anything